src/main/kotlin/no/nav/familie/ef/personhendelse/handler/PersonhendelseService.kt

```diff
@@ -62,7 +62,7 @@ class PersonhendelseService(
     }
 
     private fun handlePersonhendelse(handler: PersonhendelseHandler, personhendelse: Personhendelse, personIdent: String) {
-        if (personhendelse.skalOpphøreEllerKorrigeres()) {
+        if (personhendelse.skalOpphøreEllerKorrigeres() && personhendelse.erIkkeSivilstandHendelse()) {
```

Comment:
Skal vi ikke håndtere opphør/korrigering av sivilstand i det hele tatt?

Reply:
Sant, må nok kun lage unntak for sivilstand hendelse og at det er opphør. Dette er rettet.